client/src/Components/VideoList.js

```diff
@@ -0,0 +1,53 @@
+import OneVideoCard from "./OneVideoCard";
+const VideoList = ({ keyword, videoCards, setVideoCards }) => {
+  // const handleDelete = (idVideo) => {
+  //   setVideoCards(videoCards.filter((video) => video.id !== idVideo));
+  // };
+  const handleDelete = async (id) => {
+    try {
+      const response = await fetch(
+        `https://video-recomendations-server.onrender.com/videos/${id}`,
+        {
+          method: "DELETE",
+        }
+      );
+      if (!response.ok) {
+        throw new Error("Video not found!");
+      }
+      setVideoCards(videoCards.filter((video) => video.id !== id));
+    } catch (error) {
+      console.log(error.message);
+    }
+  };
+  return (
+    <>
+      <section className="p-5" id="recommendations">
+        <div className="container">
+          <div className="row row-cols-auto text-center custom-gap ">
+            {videoCards
+              .filter((video) => {
```

Comment:
Filter accepts a callback function which returns a truthy value - I'd ordinarily expect the callback function to return `true`/`false` rather than `video`/`undefined`...